src/main/java/joi/utils/ToDo.java

```diff
@@ -0,0 +1,30 @@
+package joi.utils;
+
+public class ToDo extends Task {
+    public ToDo (String description) throws InvalidCommandException {
+        super();
+        if (description.length() <= 5) {
+            throw new InvalidCommandException("The description of a todo cannot be empty.");
+        }
+        description = description.substring(5);
+
+        this.description = description;
+    }
+
+    public ToDo(String description, boolean status) {
+        super();
+
+        this.description = description;
+        this.isDone = status;
+    }
+
```

Comment:
Maybe inheritance can be used more powerfully by handling description and status in Task constructor?